자리수-더하기.js

```diff
@@ -0,0 +1,14 @@
+function solution(n)
+{
+    let answer = 0;
+    let isString = String(n);
+    
+    for (let i = 0; i < isString.length; i++){
+        answer += parseInt(isString[i]);
+    }
+
+    
+
+    return answer;
+}
+
```

Comment:
마찬가지로 변하지 않는 변수는 const로 선언!
컨벤션까지 신경써주세요~

```suggestion
function solution(n) {
    const isString = String(n);
    let answer = 0;
    
    for (let i = 0; i < isString.length; i++) {
        answer += parseInt(isString[i]);
    }
    return answer;
}

```